.circleci/config.yml

```
@@ -188,7 +188,7 @@ jobs:
           DB_PASSWORD: pass
           DB_PORT: 5432
       # services
-      - image: cimg/postgres:12.10
+      - image: cimg/postgres:13.16
```

What do you think to just pin to the major version ?
```suggestion
      - image: cimg/postgres:13
```

---

I tried your proposal at first, but it failed.
There is no image with only major version: https://circleci.com/developer/images/image/cimg/postgres#image-tags